mobile/lib/screens/profile/profile_widgets.dart

```diff
@@ -169,7 +169,7 @@ class SignUpSection extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.center,
         children: [
           const SizedBox(
-            height: 48,
```

Comment:
@katendejericho5 Some context on why the size of the sign-up widgets is being changed.